src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/models/CCJContent.java

```diff
@@ -47,6 +51,8 @@ public CCJContent(Map<String, Object> claim,
         Optional<StatementOfTruth> optionalStatementOfTruth = countyCourtJudgment.getStatementOfTruth();
         this.signerName = optionalStatementOfTruth.map((StatementOfTruth::getSignerName)).orElse(null);
         this.signerRole = optionalStatementOfTruth.map((StatementOfTruth::getSignerRole)).orElse(null);
+        //TODO: to be populate upon merge with ROC-4206
+        //this.dateOfOrder = formatDate(claim.countyCourtJudgmentIssuedAt)
```

Comment:
Please implement accordingly.

If before 1600 hours and next working day, taking into account bank holidays if after 1600 hours. To be picked up from countyCourtJudgmentIssuedAt (CCJ object